velero.yaml

```diff
@@ -65,11 +66,12 @@ pipeline:
   - uses: strip
 
 subpackages:
-  - name: velero-restore-helper
+  - name: "velero-restore-helper"
     description: Velero Restore Helper
     pipeline:
-      - working-directory: "${{targets.subpkgdir}}/usr/bin/"
-      - runs: mv ${{targets.destdir}}/usr/bin/velero-restore-helper .
+      - working-directory: ${{targets.subpkgdir}}/usr/bin
+        runs: |
+          mv ${{targets.destdir}}/usr/bin/velero-restore-helper  velereo-restore-helper
```

Comment:
does this need to be 

```
mv ${{targets.destdir}}/usr/bin/velero-restore-helper ${{targets.subpkgdir}}/usr/bin/
```

Reply:
its already in the `working directory`  ${{targets.subpkgdir}}/usr/bin

```
/work/work # apk info -L velero-restore-helper
velero-restore-helper-1.11.1-r0 contains:
usr/bin/velereo-restore-helper
var/lib/db/sbom/velero-restore-helper-1.11.1-r0.spdx.json
````